test/Concurrency/actor_inout_isolation.swift

```diff
@@ -131,3 +131,23 @@ extension TestActor {
     await other.modify(&value2)
   }
 }
+
+// Verify global actor protection
+
+@globalActor
+struct MyGlobalActor {
+  static let shared = TestActor()
+}
+
```

Comment:
Can we add a positive / OK test where one `@MyGlobalActor` passes state `inout` to another function `@MyGlobalActor` sync function? (I assume that we're allowing this)

Reply:
Can do!